<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportObjectInfoDialog</class>
 <widget class="QDialog" name="ExportObjectInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="generalBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>504</width>
        <height>377</height>
       </rect>
      </property>
      <attribute name="label">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0">
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="exportPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The location and filename for the dataset&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="choosePath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Choose the location and filename for the dataset</string>
         </property>
         <property name="text">
          <string>Choose File</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="fileFormat"/>
       </item>
       <item row="6" column="0" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="featureBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>504</width>
        <height>377</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Features</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QPushButton" name="selectAllFeatures">
         <property name="toolTip">
          <string>Select all features</string>
         </property>
         <property name="text">
          <string>All</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="selectNoFeatures">
         <property name="toolTip">
          <string>Clear selection</string>
         </property>
         <property name="text">
          <string>None</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTreeWidget" name="featureView">
         <property name="toolTip">
          <string>Select additional features to export. Default Features will always be exported.</string>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <column>
          <property name="text">
           <string notr="true">Select Features</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settingsBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>504</width>
        <height>377</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="toolTip">
      <string/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportObjectInfoDialog</receiver>
   <slot>validate_before_exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportObjectInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectAllFeatures</sender>
   <signal>clicked()</signal>
   <receiver>ExportObjectInfoDialog</receiver>
   <slot>select_all_features()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectNoFeatures</sender>
   <signal>clicked()</signal>
   <receiver>ExportObjectInfoDialog</receiver>
   <slot>select_no_features()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>choosePath</sender>
   <signal>clicked()</signal>
   <receiver>ExportObjectInfoDialog</receiver>
   <slot>choose_path()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileFormat</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ExportObjectInfoDialog</receiver>
   <slot>file_format_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select_all_features()</slot>
  <slot>select_no_features()</slot>
  <slot>validate_before_exit()</slot>
  <slot>choose_path()</slot>
  <slot>include_raw_changed(int)</slot>
  <slot>change_compression(QString)</slot>
  <slot>file_format_changed(int)</slot>
 </slots>
</ui>
